CodeGeneration/Sources/SyntaxSupport/Node.swift

```diff
@@ -270,4 +270,21 @@ public struct CollectionNode {
       return choices
     }
   }
+
+  public var grammar: SwiftSyntax.Trivia {
+    let grammar: String
+    if let onlyElement = elementChoices.only {
+      grammar = "``\(onlyElement.syntaxType)``*"
+    } else {
+      grammar = "(\(elementChoices.map { "``\($0.syntaxType)``" }.joined(separator: " | "))) `*`"
```

Comment:
Could make them a little more consistent here, ie. either "``foo``*" and "(``foo`` | ``bar``)*" or "``foo`` `*`" and "(``foo`` | ``bar``) `*`"

Reply:
Thanks for catching this.